enhancements/console/custom-logos.md

```diff
@@ -0,0 +1,206 @@
+          key: logo-light.svg
+          name: masthead-logo-light
+      - type: Dark
+        file:
+          key: logo-dark.svg
+          name: masthead-logo-dark
+  - type: Favicon
+    themes:
+      - type: Light
+        file:
+          key: favicon-light.png
+          name: favicon-logo-light
+      - type: Dark
+        file:
+          key: favicon-dark.png
+          name: favicon-logo-dark
+```
+
+
+### Workflow Description
+
+├── spec
+│   ├── customization
+│       ├── customLogos
+│           ├── type
+│           ├── themes
+│               ├── type
+│               ├── file
+│                   ├── key
+│                   ├── name
+└── ...
+
+### API Extensions
+
+N/A
+
+### Risks and Mitigations
+
+1. Users could set both the `CustomLogoFile` and `CustomLogos` APIs. The `CustomLogos` API configuration will take precedence over the old `CustomLogoFile` field.
+
+2. Each of the Console supported themes can be configured individually by setting either the Light or Dark theme type or by applying a default theme to all supported themes using the Default theme type. If the Default theme type is set along with a specific Dark or Light theme, the specific theme setting will override the default one.
+
+4. Users might experience confusion with the introduction of new logo configuration options, especially if they are familiar with using previous method, represented by the `CustomLogo` field, which will be deprecated. Provide a comprehensive documentation that guide users through the transition. Include clear instructions about the changes and their benefits.
+
+
+### Drawbacks
+
+N/A
+
+### Attributes Description
+
+#### customLogos
+- `type`: Enumerate which specifies the type of custom logo. Available custom logo types are `Masthead` and `Favicon`.
+- `themes`: A list of themes for which the custom logo is defined.
+
+#### themes
+- `type`: Enumerate which specifies the type of theme. Available theme types are `Light`, `Dark` and `Default`.
+- `file`: Contains the file details for the custom logo.
+
+#### file
+- `key`: The key or path to the custom logo file.
+- `name`: The name of the ConfigMap containing the custom logo file.
+
+## Test Plan
+
+Provide tests as part of the console `CustomLogos` implementation and verify that it was shown in the UI. The following tests will be added:
+ - Unit tests for API
+ - Unit and E2E tests for console-operator
+ - E2E tests for console
+
+
+## Graduation Criteria
+
+N/A
+
+
+#### Dev Preview -> Tech Preview
+
+N/A
+
+### Tech Preview -> GA
+
+N/A
+
+### Dev Preview -> Tech Preview
+
+N/A
+
+#### Removing a deprecated feature
+
+The current custom logo field in `customization.customLogo` is deprecated and will be removed at some point in the future. Users are encouraged to transition to the new custom logos configuration that supports light and dark theme modes for the masthead and favicon. The new custom logos feature also includes support for a default theme for all unspecified themes.
```

Comment:
Is it possible to say deprecated with a future release? I think it sounds a little more concrete than at some point in the future. It would make it consistent with what we use across the docs too.